plugins/vli/fu-vli-common.c

```diff
@@ -174,6 +180,8 @@ fu_vli_common_device_kind_get_size (FuVliDeviceKind device_kind)
 		return 0x20000 * 2;
 	if (device_kind == FU_VLI_DEVICE_KIND_PS186)
 		return 0x40000;
+	if (device_kind == FU_VLI_DEVICE_KIND_HYPER)
+		return 0x20000 * 2;
```

Comment:
Is hyper the deice being programmed over i2c? it's normally the "chip ID" of the i2c thing we use here e.g. PS186 rather than PARADE. If it's just a shared PD I don't think you need the FU_VLI_DEVICE_KIND_HYPER define at all.

Reply:
We're not sure what line up Hyper has in mind for the future, but I believe it's safe to remove it since our intention was to cover as many bases --just in case.

Reply:
Okay, I've removed it; I think it's confusing to have an enum defined that's not actually used.